mautic/mautic
# Fix send email to contact If send email to user action do not choose user

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no




#### Description:
When creating a form with "send email to user" action without choose user, then email is send to contact



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. create a form
3. add send email to user action
4. Choose email but don't choose use
5. Submit form and check you as contact receive email
6. After PR you have to choose user in Send email to user action
